suite-native/module-dev-utils/src/screens/DevUtilsScreen.tsx

```diff
@@ -25,31 +26,36 @@ export const DevUtilsScreen = ({
 
     return (
         <Screen header={<ScreenHeader content="DEV utils" hasGoBackIcon />}>
-            <Card>
-                <VStack spacing="medium">
-                    {!isDebugEnv() && <BuildInfo />}
-                    {isDebugEnv() && (
-                        <Button onPress={() => navigation.navigate(DevUtilsStackRoutes.Demo)}>
-                            See Component Demo
+            <VStack>
+                <Card>
+                    <VStack spacing="medium">
+                        {!isDebugEnv() && <BuildInfo />}
+                        {isDebugEnv() && (
+                            <Button onPress={() => navigation.navigate(DevUtilsStackRoutes.Demo)}>
+                                See Component Demo
```

Comment:
If you do translations for this screen, do them in all of your changed code please

Reply:
The `See Component Demo` will be never accessible to users, so I didn't translate that one. 

But I translated other buttons of the screen in 887cf49ede.